docs/configuration.md

```diff
@@ -1760,6 +1760,17 @@ Apart from these, the following properties are also available, and may be useful
   </td>
   <td>2.3.0</td>
 </tr>
+<tr>
+  <td><code>spark.io.compression.zstd.workers</code></td>
+  <td>8</td>
```

Comment:
nit: inconsistent

Reply:
Thanks, I will push it along with the benchmarking results. However, I don't have sufficient Google Analytics resources for frequent git-push.. It turns out I don't have sufficient GA resources for frequent git-push.